nss-day-cohort-45/nutyshell-talking-with-a-full-mouth-1
# Koh initial branch

# Description
The branch would let the user view their upcoming events as well as add a new event to the list or delete an already listed event. 

Note: I will still need to add the userId to each event entry and make the check weather button functional. 

## Type of change
Please delete options that are not relevant.
- [ ✅ ] New feature (non-breaking change which adds functionality)

# Testing Instructions
1. Ensure the current list of events and New Event button load to the page.
2. Click New Event button and fill out form to create new event.
3. Save your new event. (Form should reset to blank.)
4. Close event form box using the close button.
5. New event should be listed in the event list in order of date. 
6. Check that new event is also listed in your local database.
7. Delete and event. Check that it has been removed from the DOM as well as your database.

# Checklist:
- [ ✅ ] My code follows the style guidelines of this project
- [ ✅ ] I have performed a self-review of my own code
- [ ✅ ] I have commented my code, particularly in hard-to-understand areas
- [ ✅ ] My changes generate no new warnings or errors
- [ ✅ ] I have added test instructions that prove my fix is effective or that my feature works
